### ISSUE TITLE
Fix training scripts

### ISSUE THREAD

**Issue by gpauloski** [gpauloski/kfac-pytorch#3]
Imagenet:
- adjust lr does not work with kfac
- change tqdm to one from cifar

Cifar:
- no learning rate warmup

Both:
- refactor redundant code
